skimage/segmentation/slic_superpixels.py

```diff
@@ -136,9 +136,7 @@ def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
     elif isinstance(sigma, (list, tuple)):
         sigma = np.array(sigma, dtype=np.double)
     if (sigma > 0).any():
-        # add zero smoothing for multichannel dimension
-        sigma = list(sigma) + [0]
-        image = ndi.gaussian_filter(image, sigma)
+        image = gaussian(image, sigma, multichannel=True)
```

Comment:
The previous line explicitly added a final zero to `sigma`, which is what `multichannel=True` does in our `gaussian`. There is no change in the way this function operated.